Egregious-Eggnog/reactive-nutshell
# added test data to example database.json and erd to readme

# Description

Added test data to example database.json and the ERD to the end of the README. Didn't update anything else in the README so we can continue to use as a reference.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.


# Testing Instructions

Copy the `database.json.example` file as `database.json` and check the data with json server


# Checklist:
